component-samples/demo/owner/service.yml

```diff
@@ -10,11 +10,7 @@ system-properties:
   app-data.dir: ./app-data
   application.version: 1.1-SNAPSHOT
   service.name: "Owner"
-  http.proxyHost: $(http_host)
-  http.proxyPort: $(http_port)
-  https.proxyHost: $(https_host)
```

Comment:
Added this property for SVI resource related proxy setting. Consider adding it back.